rust-wasm-id-allocator/distributed-id-allocator/src/compressor.rs

```diff
@@ -21,12 +21,13 @@ pub struct IdCompressor {
 }
 
 impl IdCompressor {
+    #[cfg(feature = "uuid-generation")]
     pub fn new() -> Self {
         let session_id = SessionId::new();
         IdCompressor::new_with_session_id(session_id)
     }
 
-    pub(crate) fn new_with_session_id(session_id: SessionId) -> Self {
+    pub fn new_with_session_id(session_id: SessionId) -> Self {
```

Comment:
Had to make this public, I added it to the type exposure issue as a thing to move into the third private crate.